src/epiviz.gl/webgl-vis.js

```diff
@@ -303,6 +298,16 @@ class WebGLVis {
     this.parent.addEventListener(type, listener, options);
   }
 
+  /**
+   * Dispatches an event on the visualization on the appropriate component.
+   * @param {String} eventName
```

Comment:
this is a controlled vocabulary of event names we support and should probably reflect that in the documentation

Reply:
Thank you, Updated.